### ISSUE TITLE
About The Text Orientation when Fine-Tuning Arabic Recognition Model

### ISSUE THREAD

**Issue by MohieEldinMuhammad** [PaddlePaddle/PaddleOCR#9922]
I'm fine-tuning arabic_PP-OCRv3_rec_train model and when I resume training from the best_accuracy model the accuracy always starts with zero and the overall accuracy of the model after training drops significantly, so I'm suspecting that there is a problem in the orientation the Arabic is red with it while comparing the predictions with the ground truth text. Can anyone help me with this?
